packages/fetchai/connections/p2p_noise/connection.yaml

```diff
@@ -0,0 +1,27 @@
+name: p2p_noise
+author: fetchai
+version: 0.1.0
+description: The p2p noise connection implements an interface to standalone golang
+  noise node that can exchange aea envelopes with other agents participating in the
+  same p2p network.
+license: Apache-2.0
+aea_version: '>=0.3.0, <0.4.0'
+fingerprint:
+  __init__.py: QmbPzrjd27coFfS2vN9xDL4uARKZWCCmtWvmEuzGKSjxf7
+  aea/api.go: QmYYRrMemRA1x9UsSg2VrofprHyH5oZv9S3FrF6G9jzCC1
+  connection.py: QmaayZzvz5WY6D5SD8MDwqRoscxrxB8fQfUDy3S8JG84Eu
+  noise_node.go: QmUf63vTTndzgqDeogsQ3fEERtYJ5hQotgiWgb7rtqMSzZ
+  publickey.py: QmPyfGWQJAhHV1SQoNiCkS5uDhFXSJfPrPe7igQ1uxcPsa
+fingerprint_ignore_patterns: []
+protocols: []
+class_name: P2PNoiseConnection
+config:
+  entry_peers:
```

Comment:
entry_peers is maybe not very explanatory? How about seed_nodes or something like that?

Reply:
I don't know if seed_nodes is any better. For now I change it to `noise_entry_peers`